.github/workflows/deployments.yml

```diff
@@ -16,6 +16,7 @@ jobs:
       matrix:
         network: [testnet]
     env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
u need to confirm if this will work. I don't know if the token permissions are shared across repos

Reply:
this is fed by the github runner

Reply:
yep, but you are trying to use the token to get artifacts from `kakarot-ssj` right?